### PR TITLE
feat(tldraw): init

### PR DESCRIPTION
**Pull request by GenShibe**
## 🎉 Theme for tldraw 🎉
a drawing website with an infinite canvas

frappe

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `preview.webp` - composite image of all four individual flavor screenshots stitched together,
    generated via [Catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk#readme).

### CODE REVIEW

**Review by isabelroses — APPROVED** [review]

**Review by uncenter — CHANGES_REQUESTED** [review]
As we discussed in DMs I think the paint/SVG strokes themselves on the canvas should be themed as well.

**Review by uncenter — CHANGES_REQUESTED** [review]
Instead of `darken(@color, 60%)` we should be using something like `fade(desaturate(@color, 10%), 30%)`. That more correctly matches what the inside of shapes looks like for SVGs.

**Comment by uncenter** [comment]
| Current | Suggested |
| --- | --- |
| Screenshot 2024-03-14 at 20 34 29 (Arc) |  Screenshot 2024-03-14 at 20 34 07 (Arc) |

**Comment by uncenter** [comment]
"Blue" and "light blue" seem to be switched for the pen mode (but not the shapes):

https://github.com/catppuccin/userstyles/assets/47499684/9de5b504-486f-4511-a9b5-c6bcb6f80751

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -683,6 +683,13 @@ userstyles:
     readme:
       app-link: "https://tabnews.com.br"
       current-maintainers: [*Guaxinim5573]
+  tldraw:
+    name: tldraw
+    categories: [productivity]
+    color: text
+    readme:
+      app-link: "https://www.tldraw.com"
+      current-maintainers: [*genshibe]
```
Comment (isabelroses) [inline scripts/userstyles.yml:692]: might be worth noting that the exports are not in the catppuccin theme in the FAQ section

Comment (uncenter) [reply scripts/userstyles.yml:692]:
```suggestion
    readme:
      app-link: "https://www.tldraw.com"
      usage: |+
        > [!NOTE]
        > This theme only themes the colors on the tldraw website. Exported graphics **will not be Catppuccin-themed**.
      current-maintainers: [*genshibe]
```

> [!NOTE]
> This theme only themes the colors on the tldraw website. Exported graphics **will not be Catppuccin-themed**.

**Review by uncenter — CHANGES_REQUESTED** [review]
Screenshot 2024-03-15 at 13 47 13 (CleanShot)

Instead of black and white these should be white and gray (to reflect what is actually drawn on the canvas and what it looks like without the theme).

**Comment by uncenter** [comment]
Also, selected buttons/options from dropdowns and such have unthemed gradients:

Screenshot 2024-03-15 at 13 49 50 (Arc)

The following CSS is what affects all of these elements as far as I can tell, so if you can theme this that should be everything :p
```css
.tlui-menu__submenu__trigger[data-state=open]:not(:hover):after {
    border-radius: var(--radius-1);
    background: linear-gradient(90deg,rgba(144,144,144,0) 0%,var(--color-muted-2) 100%);
}
.tlui-menu-zone *[data-state=open]:after {
    background: linear-gradient(180deg,rgba(144,144,144,0) 0%,var(--color-muted-2) 100%);
}
```

**Review by isabelroses — APPROVED** [review]

**Comment by uncenter** [comment]
Still some things I have to fix, don't merge yet!

**Comment by uncenter** [comment]
- [ ] Interior/literal fill color of shapes and sticky notes

**Comment by uncenter** [comment]
- Shape fill
  - Solid
  - Pattern

**Comment by uncenter** [comment]
I think this is in a mergeable state. We can open an issue to track the unthemed fill colors of shapes with solid pattern. @GenShibe do you want to mark this as ready for review?

**Review by uncenter — APPROVED** [review]

**Pull request merged**